lib/xdrgen/generators/java.rb

```diff
@@ -637,8 +643,6 @@ def encode_type(type, value)
           raise "cannot render quadruple in golang"
         when AST::Typespecs::Bool ;
           "stream.writeInt(#{value} ? 1 : 0)"
-        when AST::Typespecs::String ;
-          "stream.writeString(#{value})"
```

Comment:
With this change the `writeString` and `readString` functions become unused I think. Should we continue to use them and redefine inside those functions how we write and read the string? Or should we remove those functions?

https://github.com/stellar/xdrgen/blob/c683684abb3cb2c8b8c3dc12dc7c6cc9bf1e3b60/lib/xdrgen/generators/java/XdrDataOutputStream.erb#L17-L21

https://github.com/stellar/xdrgen/blob/c683684abb3cb2c8b8c3dc12dc7c6cc9bf1e3b60/lib/xdrgen/generators/java/XdrDataInputStream.erb#L24-L29

Reply:
I pushed a commit which removes the methods